http/python/requests/http_requests/requests_request_adapter.py

```diff
@@ -331,11 +328,11 @@ def get_request_from_request_information(
         self, request_info: RequestInformation
     ) -> requests.PreparedRequest:
         req = requests.Request(
-            method=str(request_info.http_method),
-            url=request_info.get_url(),
-            headers=request_info.get_request_headers(),
+            method=request_info.http_method.value,
+            url=request_info.url,
+            headers=request_info.request_headers,
             data=request_info.content,
-            params=request_info.query_parameters,
+            # params=request_info.query_parameters,
```

Comment:
To delete or not to delete this line?

Reply:
Deleted in https://github.com/microsoft/kiota/pull/1761/commits/fe4685cb1a95dde69b5a23d5cb348430ecc901b7 as we are already appending query parameters to the url do not need the default implementation to pass query parameters.